vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -95,11 +95,22 @@
       </ComplexType>
 
       <ComplexType Name="CollectionKindSnapshot" BaseType="Temporal.CollectionKindType">
-        <Annotation Term="Core.Description" String="Each OData entity corresponds to a temporal object and by default only shows data of the current time slice" />
+        <Annotation Term="Core.Description" String="Each OData entity represents a snapshot of data at a point in application time that is the same for all entities in the collection" />
+        <Annotation Term="Core.LongDescription">
+          <String>The point in application time is defined by the `$at` clause of a request; default is the current point in time.</String>
+        </Annotation>
       </ComplexType>
 
       <ComplexType Name="CollectionKindTimeSlice" BaseType="Temporal.CollectionKindType">
-        <Annotation Term="Core.Description" String="Each OData entity corresponds to a time slice and all time slices of a temporal object are part of the collection" />
+        <Annotation Term="Core.Description" String="Each OData entity represents data during an interval of application time; it is called a *time slice*" />
+        <Annotation Term="Core.LongDescription">
+          <String>An entity in a temporal collection of this kind and an instance of the same entity type are said to *overlap*
+          if the intervals defined by their `TimeSliceStart` and `TimeSliceEnd` properties overlap
+          and their other ("non-temporal") key properties are identical.
+          A temporal collection of this kind MUST NOT contain overlapping entities from the same entity set;

```

Comment:
This is necessary because the entities `EntitySet('A')/ContainmentNavProp(1)` and `EntitySet('B')/ContainmentNavProp(1)` have the same key property `1` and still are distinct because they live in different implicit entity sets.

Reply:
The part after the semicolon replaces the following:
https://github.com/oasis-tcs/odata-vocabularies/blob/e6f7b53c199d88f185cb4e375bc8167ca8323508/vocabularies/Org.OData.Temporal.V1.xml#L102